generator/config/aggregation-stages/collStats.yaml

```diff
@@ -0,0 +1,15 @@
+# $schema: ../schema.json
+name: $collStats
+category:
+    - Stage
+link: 'https://www.mongodb.com/docs/manual/reference/operator/aggregation/collStats/'
+type:
+    - Stage
+encode: single
+description: |
+    Returns statistics regarding a collection or view.
+arguments:
+    -
+        name: config
+        type:
+            - object
```

Comment:
I did not modelize the full object. Let's use `object` function for that. People will have to read the documentation anyway.
I could have made it a variadic map.